linux/README.md

```diff
@@ -62,8 +62,8 @@ _src/packageTest/java/packaging_ on them.
 
 ### Build a Debian specific package for a version
 
-- replace `<version>` with `8|11|17|19`
-- replace `<vendor>` with `temurin|dragonwell`
+- replace `<version>` with `8|11|17|19|20|21|22|23`
+- replace `<vendor>` with `temurin|dragonwell|microsoft`
```

Comment:
Not openj9 here as well?

Reply:
@karianna this README section is under the `build debian` section. Since openj9 only has config files up for `redhat` `RPMs`, this would mean that `openj9` as a vendor for `debian` installers is currently not supported by this repo